src/app/app-pipes/duration.pipe.ts

```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/** Formats given duration in minutes and outputs as hh h mm m. In case of duration less than 1 hr displays only minutes. */
+@Pipe({
+  name: 'duration'
+})
+export class DurationPipe implements PipeTransform {
+  transform(duration: number, ...args: never[]): any {
```

Comment:
I'm curious, why the `...args`?

Reply:
Nothing in particular, I wanted to "keep" the code generated from ng cli and maybe extend this pipe to accept any other arguments in the future. This could be removed without side effects, though.